specs/electra/beacon-chain.md

```diff
@@ -631,12 +634,19 @@ def switch_to_compounding_validator(state: BeaconState, index: ValidatorIndex) -
 
 ```python
 def queue_excess_active_balance(state: BeaconState, index: ValidatorIndex) -> None:
+    validator = state.validators[index]
     balance = state.balances[index]
     if balance > MIN_ACTIVATION_BALANCE:
         excess_balance = balance - MIN_ACTIVATION_BALANCE
         state.balances[index] = MIN_ACTIVATION_BALANCE
-        state.pending_balance_deposits.append(
-            PendingBalanceDeposit(index=index, amount=excess_balance)
+        state.pending_deposits.append(
+            PendingDeposit(
+                epoch=get_current_epoch(state),
+                pubkey=validator.pubkey,
+                withdrawal_credentials=validator.withdrawal_credentials,
+                amount=balance,
+                signature=BLSSignature(),
```

Comment:
For `SyncAggregate`, `G2_POINT_AT_INFINITY` is used instead of a zero value when noone signed.

Or, `None` with EIP-7688 optionals (but I guess zero value or infinity is also fine, as it is not checked anyway).